iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -81,15 +103,20 @@ class ConversationMessagesCell: UICollectionViewCell {
     }
 
     /// Configures the cell to display the given messages. The message sequence should be ordered newest to oldest.
-    func set(sequence: MessageSequence) {
-        self.conversation = sequence
+    func set(conversation: Conversation) {
+        // Create a new conversation controller if this is a different conversation than before.
+        if conversation.cid != self.conversation?.cid {
+            self.conversationController = ChatClient.shared.channelController(for: conversation.cid)
+            self.subscribeToUpdates()
+        }
```

Comment:
This cell is less generic now that it doesn't take a message sequence, but it makes it easier to set up the conversation controller.

In the future, we could make a MessageSequenceController protocol that works for both conversations and messages.